src/main/java/Sirius.java

```diff
@@ -0,0 +1,148 @@
+                break;
+            default:
+                list.add(new Todo(taskName, false));
+        }
+        System.out.println("Got it. I've added this task:");
+        System.out.println(list.get(taskCounter).toString());
+        taskCounter++;
+        System.out.println("Now you have " + taskCounter + " tasks in the list.");
+        System.out.println(SAPERATOR);
+    }
+    public static void listTasks(ArrayList<Task> list){
+        System.out.println(SAPERATOR);
+        System.out.println("Here are the tasks in your list:");
+        for (int i = 0; i < taskCounter; i++) {
+            System.out.print(i + 1 + ". ");
+            System.out.println(list.get(i).toString());
+        }
+        System.out.println(SAPERATOR);
+    }
+    public static String[] splitCommand(String userInput){
+        String[] slashCommand = userInput.split(SLASH);
+        String commandContent = slashCommand[0];
+        String commandPrefix = commandContent.split(SPACE)[0];
+        String taskName = commandContent.replace(commandPrefix, EMPTY).trim();
+
+        if (slashCommand.length == 1){ //Todo
+            String[] commandPieces = new String[2];
+            commandPieces[0] = commandPrefix;
+            commandPieces[1] = taskName;
+            return commandPieces;
+        }
+        else if (slashCommand.length == 2){ //Deadline
+            String[] commandPieces = new String[3];
+            commandPieces[0] = commandPrefix;
+            commandPieces[1] = taskName;
+            commandPieces[2] = slashCommand[1].replace("by", EMPTY).trim();
+            return commandPieces;
+        }
+        else if (slashCommand.length == 3){ //Event
+            String[] commandPieces = new String[4];
+            commandPieces[0] = commandPrefix;
+            commandPieces[1] = taskName;
+            commandPieces[2] = slashCommand[1].replace("from", EMPTY).trim();
+            commandPieces[3] = slashCommand[2].replace("to", EMPTY).trim();
+            return commandPieces;
+        }
+        else{
+            return new String[0];
+        }
+    }
+
+    public static void main(String[] args) {
+        sayHello();
+        Scanner scanner = new Scanner(System.in);
+        while (isExit) {
+            String userInput = scanner.nextLine();
+            String[] commandPieces = splitCommand(userInput);
+            String commandPrefix = commandPieces[0];
+
+            switch (commandPrefix) {
+                case BYE:
+                    sayGoodbye();
+                    break;
+                case LIST:
+                    listTasks(list);
+                    break;
+                case MARK:
+                    markTask(commandPieces, list, true);
+                    break;
+                case UNMARK:
+                    markTask(commandPieces, list, false);
+                    break;
+                default:
+                    addTask(commandPieces, list);
+            }
```

Comment:
I like that you used SLAP hard (coding practice) into this part (I personally didn't) and you used switch instead of else if (it was unexpected to me). Good job applying what you learnt in class!